addons/coupon/data/expire_coupon_cron.xml

```diff
@@ -12,5 +12,13 @@
             <field name="interval_type">days</field>
             <field name="numbercall">-1</field>
         </record>
+        <record id="barcode_rule_coupon" model="barcode.rule">
```

Comment:
leave it in the pos coup, you cannot put it here, beacause coupon has no dependencies to barcode

Reply:
Oh true, it doesn't depend on barcode. XD